src/FSharp.Data.GraphQL.Server/Values.fs

```diff
@@ -163,14 +188,27 @@ let rec internal compileByType (inputObjectPath: FieldPath) (inputSource : Input
                             | ValueSome field ->
                                 match Map.tryFind field.Name props with
                                 | None -> Ok <| wrapOptionalNone param.ParameterType field.TypeDef.Type
+                                | Some input when isNull (box field.ExecuteInput) ->
+                                    // hack around the case where field.ExecuteInput is null
```

Comment:
maybe take this hack as a separate function and not a nested function inside many loops...